src/rest/entries.js

```diff
@@ -22,17 +22,46 @@ export const createUrlForEntries = projectId => (
 
 // TODO: move this somewhere else
 const ONE_DAY = 24 * 60 * 60 * 1000;
+const toDays = (date) => {
+    if (!date) {
+        return undefined;
+    }
+
+    return Math.round(new Date(date).getTime() / ONE_DAY);
+};
+
+const totMinutes = (time) => {
```

Comment:
Is *totMinutes* intentional name?

Reply:
it can mean total minutes if you want :stuck_out_tongue: